Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractBoundingBox.java

```diff
@@ -197,6 +186,4 @@ public boolean contains(int dimension, double value) {
     @Override
     public abstract boolean equals(Object other);
```

Comment:
Do we need to keep this in `AbstractBoundingBox`? the `equals` method is defined on `Object`.

Reply:
No. Can remove in next PR.